hack/run-ci-e2e-test.sh

```diff
@@ -115,6 +115,16 @@ fi
 # Enable debug logging
 WMCO_SUB=$(oc get sub -n $WMCO_DEPLOY_NAMESPACE --no-headers |awk '{print $1}')
 oc patch subscription $WMCO_SUB -n $WMCO_DEPLOY_NAMESPACE --type=merge -p '{"spec":{"config":{"env":[{"name":"ARGS","value":"--debugLogging"}]}}}'
+retries=0
+while [[ $(oc get -n $WMCO_DEPLOY_NAMESPACE pod -l name=windows-machine-config-operator -oname | wc -l) != "1"  || $(oc get -n $WMCO_DEPLOY_NAMESPACE pod -l name=windows-machine-config-operator -oyaml) != *"--debugLogging"* ]]; do
```

Comment:
could the check  `!= *"--debugLogging"*` not be working as expected?

Maybe a grep would be more readable? 
oc get pod -n $WMCO_DEPLOY_NAMESPACE -oyaml | grep -m1 "\--debugLogging"